src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -215,6 +215,7 @@ struct network {
     void configure_primitives_second_output();
     void build_insts_deps();
     uint32_t get_id() const { return net_id; }
+    uint32_t get_local_id() const { return _local_net_id; }
```

Comment:
Is this used only for model cache? Could you please add some comment of definition of _local_net_id for future reference?

Reply:
Yes, it is used only for model caching. I added comments for the `_local_net_id` as you reviewed. Thank you.